batch-setup/provision.sh

```diff
@@ -97,6 +97,7 @@ export METATILE_SIZE='%(metatile_size)d'
 
 export NUM_DB_REPLICAS='%(num_db_replicas)d'
 export MAX_VCPUS='%(max_vcpus)d'
+export RUN_POST_IMPORT_STEPS='%(run_post_import_steps)s'
```

Comment:
I'm assuming this comes in as a Python attempt at converting a bool into a string so it should be "True" or "False"... right?

Reply:
how is run_post_import_steps get its value? maybe you need to edit this dict https://github.com/tilezen/tileops/blob/94ca72863e36940a0b2b9dd5bf96ecdeb44eecfb/batch-setup/cron.py#L607

Reply:
Yes, `$(run_post_import_step)s` will stringify your boolean into `True` or `False`.

Reply:
@peitili - I did edit the dict!